docs/release-process.md

```diff
@@ -0,0 +1,173 @@
+# How to release c2pa-rs and related crates
+
+The build process for the crates in this repo is automated using [release-plz](https://release-plz.dev), a Rust-specific CI tool. Release-plz is two components: a Rust-based command-line tool ([GitHub](https://github.com/release-plz/release-plz)) and a pre-packaged GitHub Actions wrapper ([GitHub](https://github.com/release-plz/action)). We use the pre-packaged GitHub Actions wrapper in this project. Colin and Eric have both submitted fixes to release-plz. The developer is generally responsive and PRs are typically merged and released within a week or two.
+
+Release-plz leverages information contains in the various `Cargo.toml` files and its own [`release-plz.toml`](../release-plz.toml) configuration file. In this document, we will talk only about how it release-plz is configured for this project. For more detail about the available settings in release-plz, we recommend reading the [release-plz documentation site](https://release-plz.dev/docs), which is quite extensive.
+
+## How it works
+
+We use the pre-packaged [release-plz GitHub Actions wrapper](https://github.com/release-plz/action) to trigger builds. This is configured in our [`release.yml`](../.github/workflows/release.yml) task.
+
+For additional information, please read the [release-plz documentation for GitHub Action](https://release-plz.dev/docs/github/quickstart).
+
+In this repo, the build automation is triggered for each commit to the `main` branch.
+
+This task runs using a GitHub access token tied to Eric's `@scouten-adobe` account, which is why all PRs and releases are under Eric's name. (See ticket CAI-7621 for moving this to a team-owned bot account.)
```

Comment:
Hopefully there a high priority on getting this bot account. With release-plz submitting PRs it is getting very hard to know which is from Eric and which is automation.